docs/misc/config.md

```diff
@@ -15,16 +15,27 @@ A configuration can be created using a subtype of `IConfigSpec`. NeoForge implem
 `ModConfigSpec.Builder#configure` is typically used with a `static` block and a class that takes in `ModConfigSpec.Builder` as part of its constructor to attach and hold the values:
 
 ```java
-// In some config class
-ExampleConfig(ModConfigSpec.Builder builder) {
-    // Define values here in final fields
-}
+//Store the config properties as finals
+final ModConfigSpec.ConfigValue<String> testValue;
 
-// Somewhere the constructor is accessible
+public ExampleConfig(ModConfigSpec.Builder builder) {
+    //Define each property
+    testValue = builder.define("test_value", "Hello world!");
```

Comment:
I think if you wanted to show off this better, it should be moved to the other example section and have that wrapped in a constructor. Also, `test_value` and `Hello world!` are not really clear on what they are supposed to be since it hasn't been explained at this point.